phpBB/adm/style/overall_header.html

```diff
@@ -143,6 +143,13 @@
 							</div>
 						<!-- ENDIF -->
 					</div>
-	
+
 					<div id="main">
 						<div class="main">
+							<!-- IF CONTAINER_EXCEPTION !== false -->
```

Comment:
I don't really like that you are mixing phpBB template syntax with twig syntax here. Why not just use twig for the if, too?